app/org/sagebionetworks/bridge/models/schedules/TaskScheduler.java

```diff
@@ -42,6 +44,21 @@ protected DateTime getScheduledTimeBasedOnEvent(Schedule schedule, Map<String, D
         return eventTime;
     }
     
+    protected DateTime getFirstEventDateTime(String eventIdsString, Map<String, DateTime> events) {
+        if (events == null) {
+            return null;
+        } else if (!eventIdsString.contains(",")) {
+            return events.get(eventIdsString);
+        }
+        List<String> eventIds = Arrays.asList(eventIdsString.split("\\s*,\\s*"));
+        for (String thisEventId : eventIds) {
+            if (events.get(thisEventId) != null) {
+                return events.get(thisEventId);
+            }
+        }
+        return null;
```

Comment:
This is an error in the schedule itself. What can we do here? Probably fall back to enrollment.